recipes/wayland/all/conanfile.py

```diff
@@ -3,12 +3,13 @@
 from conan.tools.build import cross_building
 from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
 from conan.tools.files import copy, get, replace_in_file, rmdir
+from conan.tools.gnu import PkgConfigDeps
 from conan.tools.layout import basic_layout
 from conan.tools.meson import Meson, MesonToolchain
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.50.0"
+required_conan_version = ">=1.52.0"
```

Comment:
Why 1.52.0 is needed?

Reply:
That's when `build_context_activated` was introduced for `PkgConfgDeps. See the PR [here](https://github.com/conan-io/conan/pull/11979). It's actually the first bullet point in the release notes for 1.52.0, too.

Reply:
Good catch!